### PR TITLE
onnxruntime fix for Python 3.9.16 in macos

### PR DESCRIPTION
**Pull request by babenek**
## Description

- workaround for issue for macos Python 3.9.16 (pip23) and latest version of onnxruntime v1.14.0
https://github.com/microsoft/onnxruntime/issues/14663

## How has this been tested?

- [x] GitHub actions pass

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,8 +1,14 @@
+import sys
+
 import setuptools
 
 with open("README.md", "r", encoding="utf8") as fh:
     long_description = fh.read()
 
+onnxruntime_pkg = "onnxruntime"
+if "darwin" == sys.platform and 9 == sys.version_info.minor:
+    onnxruntime_pkg += "<=1.13.1"
```
Comment (csh519) [inline setup.py:10]: Could you please add annotation why those lines are added?
For the history :slightly_smiling_face:

Comment (babenek) [inline setup.py:10]:
```suggestion
    # workaround for https://github.com/microsoft/onnxruntime/issues/14663
    # onnxruntime v1.14.0 for macos with Python3.9.16 has package issue (have (arm64), need (x86_64)))
    onnxruntime_pkg += "<=1.13.1"
```

Comment (babenek) [reply setup.py]: @csh519 
BTW, with #263  it is a good chance to test new release workflow

**Review by csh519 — APPROVED** [review]
Thank you for adding workaround for `onnxruntime`.

LGTM 👍

File: setup.py (hunk 1 above)
Comment (csh519) [reply setup.py]: Yup, let's see :)

**Review by iuriimet — APPROVED** [review]

**Pull request merged**